src/ui/issuecolumn/IssueColumn.java

```diff
@@ -68,6 +70,21 @@ public IssueColumn(UI ui, Stage mainStage, Model model, ColumnControl parentColu
 		this.ui = ui;
 		getChildren().add(createFilterBox());
 		setupIssueColumnDragEvents(model, columnIndex);
+		this.setOnMouseClicked(e-> {
+			ui.triggerEvent(new ColumnClickedEvent(columnIndex));
+			getParent().requestFocus();
+		});
+		focusedProperty().addListener(new ChangeListener<Boolean>() {
+			@Override
+			public void changed(ObservableValue<? extends Boolean> unused, Boolean wasFocused, Boolean isFocused) {
+				if (!isFocused) {
+					if (getStyleClass().contains("issue-panel-focused"))
+						getStyleClass().remove("issue-panel-focused");
+					return;
+				}
+				getStyleClass().add("issue-panel-focused");
+			}
+		});
 	}
```

Comment:
Maybe use an assertion? So if the style class gets on there for some reason we would be alerted.